var/spack/repos/builtin/packages/libtiff/package.py

```diff
@@ -56,16 +56,27 @@ class Libtiff(AutotoolsPackage):
     conflicts('+webp', when='@:4.0.9')
 
     # https://gitlab.com/libtiff/libtiff/-/merge_requests/243
-    patch('no-include-root.patch', when='@4.3.0')
+    patch('no-include-root.patch', when='@4.3.0 platform=darwin')
 
-    depends_on('automake', when='@4.3.0', type='build')
-    depends_on('autoconf', when='@4.3.0', type='build')
-    depends_on('libtool', when='@4.3.0', type='build')
-    depends_on('m4', when='@4.3.0', type='build')
+    depends_on('gl', when='@4.3.0 platform=darwin')
+    depends_on('glu', when='@4.3.0 platform=darwin')
+    depends_on('freeglut', when='@4.3.0 platform=darwin')
+    depends_on('libsm', when='@4.3.0 platform=darwin')
+    depends_on('libice', when='@4.3.0 platform=darwin')
+    depends_on('libxi', when='@4.3.0 platform=darwin')
+    depends_on('libx11', when='@4.3.0 platform=darwin')
+
+    depends_on('automake', when='@4.3.0 platform=darwin', type='build')
+    depends_on('autoconf', when='@4.3.0 platform=darwin', type='build')
+    depends_on('libtool', when='@4.3.0 platform=darwin', type='build')
+    depends_on('m4', when='@4.3.0 platform=darwin', type='build')
 
     @property
     def force_autoreconf(self):
-        return self.spec.satisfies('@4.3.0')
+        if self.spec.satisfies('platform=darwin'):
+            return self.spec.satisfies('@4.3.0')
+        else:
+            return False
```

Comment:
How is this different than:
```suggestion
        return self.spec.satisfies('@4.3.0 platform=darwin')
```
?

Reply:
Ha, it is not different, my brain just went in a different direction.